external/include/mbedtls/config.h

```diff
@@ -2923,12 +2925,41 @@
 
 #endif							/* MBEDTLS_LIGHT_DEVICE */
 
+#if defined(CONFIG_SE)
+#define MBEDTLS_ENABLE_HARDWARE_ALT
+
+#if defined(CONFIG_HW_RNG)
+#define MBEDTLS_ENTROPY_HARDWARE_ALT
+#endif
+
+#if defined(CONFIG_HW_DH_PARAM)
+#define MBEDTLS_DHM_ALT
+#endif
+
 #if defined(CONFIG_HW_ECDH_PARAM)
 #undef MBEDTLS_KEY_EXCHANGE_ECDH_ECDSA_ENABLED
+#define MBEDTLS_ECDH_GEN_PUBLIC_ALT
```

Comment:
Here are so many new definitions introduced just for TizenRT.
How about discerning new ones from previous ones by naming?

Reply:
Both are not introduced only for TizenRT. It is supported by the public open-source mbedtls, especially in ecdh.c file. All of newly defined configs has "CONFIG_" prefix. Thanks for the comment.